cmd/csi-resizer/main.go

```diff
@@ -44,8 +45,9 @@ var (
 	resyncPeriod = flag.Duration("resync-period", time.Minute*10, "Resync period for cache")
 	workers      = flag.Int("workers", 10, "Concurrency to process multiple resize requests")
 
-	csiAddress  = flag.String("csi-address", "/run/csi/socket", "Address of the CSI driver socket.")
-	csiTimeout  = flag.Duration("csiTimeout", 15*time.Second, "Timeout for waiting for CSI driver socket.")
+	csiAddress = flag.String("csi-address", "/run/csi/socket", "Address of the CSI driver socket.")
+	timeout    = flag.Duration("timeout", 15*time.Second, "Timeout for waiting for CSI driver socket.")
```

Comment:
How come 15? All the other sidecars are 10 seconds. Actually, do we want to default a timeout value? I find 10 seconds is quite short for real drivers. xref: https://github.com/kubernetes-csi/external-provisioner/issues/451#issuecomment-662604169

Reply:
@msau42 sure we can keep in parity between sidecars ie 10 seconds.  I kept it 15 seconds as obsoleted  `csitimeout ` default was "15s".

Reply:
ok.. I changed to 10s default @msau42 @gnufied

Reply:
Also - I kind of dislike, renaming the parameter and making it required in same release. It causes problems for downstream users. So, if we choose to make this parameter mandatory, we should do this in 2.0 IMO. 

@msau42 does 80s as a default sound good to you? The reason I picked this is because, from my experience this is "good enough" default for most in-tree plugins for which there is a equivalent CSI driver. If we all agree @humblec can you update the PR to use `80s` as default timeout?

Reply:
sure @gnufied , so once @msau42 ack on `80s` , I will change the default value to the same :+1:

Reply:
Also, I do second to `2.0.0` with this new parameter instead of changing it in the minor release.

Reply:
we spoke in CSI implementation call today. It was decided that, 10s is a good default because it exposes driver idempotency issues and specific drivers can tune this value.

Reply:
Thanks for the update @gnufied ! :+1: